helm_deploy/values-preprod.yaml

```diff
@@ -33,6 +33,7 @@ env:
   TOKENVERIFICATION_API_ENABLED: true
   REDIS_ENABLED: true
   VLBEVENT_EXPORT_SPREADSHEET_ID: "1YUKHKsq9VftSLm6twO9F172gHDh6Xp8Tj6V1HEN6FbA"
```

Comment:
maybe set this to true so we can test the toggle is working?

Reply:
So it doesn't work in dev, does work in preprod... meh...

Reply:
okay. seems fair